main/settings.py

```diff
@@ -620,7 +620,7 @@ def get_all_config_keys():
 MIDDLEWARE_FEATURE_FLAG_COOKIE_MAX_AGE_SECONDS = get_int(
     "MIDDLEWARE_FEATURE_FLAG_COOKIE_MAX_AGE_SECONDS", 60 * 60
 )
-
+SEARCH_PAGE_CACHE_DURATION = 60 * 60 * 24
```

Comment:
Can you change this to `get_int("SEARCH_PAGE_CACHE_DURATION", 60 * 60 * 24)` so we have the option of overriding with a variable in the settings - could be useful to be able to reduce the duration quickly if we experience memory issues with the caches